fala/templates/generic-py-base.html

```diff
@@ -40,24 +40,33 @@
-            <image src="{{ static('images/moj-logotype-crest.png') }}" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
-          </svg>
-          <a href="https://www.gov.uk/government/organisations/legal-aid-agency" class="govuk-header__link govuk-header__link--homepage">
-            <span class="govuk-header__logotype">
-              <span class="govuk-header__logotype-text">
-                Legal Aid Agency
-              </span>
-            </span>
-          </a>
-        </div>
-        <div class="govuk-header__content">
-          {% block proposition %}{% endblock %}
```

Comment:
Was this block not being used before as it has now been removed from the template

Reply:
Old block, readding it adds a massive split on the page 
<img width="1491" alt="Screenshot 2024-02-19 at 10 57 50" src="https://github.com/ministryofjustice/fala/assets/65071578/b7c22c2c-f11c-4be0-b699-4e53b27737ba">